code/_onclick/hud/screen_objects.dm

```diff
@@ -124,6 +124,8 @@
 	var/selecting = "chest"
 
 /obj/screen/zone_sel/update_icon(mob/living/user)
+	if(!user)
```

Comment:
How is this admin related? Also if this is called without a valid user something is seriously wrong